addons/l10n_th/views/account_move.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="report_commercial_invoice_document" inherit_id="account.report_invoice_document" primary="True"/>
```

Comment:
I just tested it and I think it doesn't work as intended - the commercial invoice ends up being the same as the tax invoice. A solution for this would be to add a flag to all the changes made by the tax invoice, that is toggled on the commercial invoice template using a `t-set` - see my suggestion below.